mitiq/pec/representations/depolarizing.py

```diff
@@ -0,0 +1,215 @@
+# Copyright (C) 2020 Unitary Fund
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+from typing import List
+from itertools import product
+
+from cirq import Operation, X, Y, Z, Circuit
+from mitiq import QPROGRAM
+from mitiq.pec.types import OperationRepresentation, NoisyOperation
+from mitiq.conversions import convert_to_mitiq, convert_from_mitiq
+
+
+def depolarizing_representation(
+    ideal_operation: QPROGRAM, noise_level: float
+) -> OperationRepresentation:
+    r"""As described in [Temme2017]_, this function maps an
+    ``ideal_operation`` :math:`\mathcal{U}_{\beta}` into its quasi-probability
+    representation (QPR), which is a linear combination of noisy implementable
+    operations :math:`\{\eta_{\alpha} \mathcal{O}_{\alpha}\}`.
+
+    This function assumes depolarizing noise is the only noise present. In
+    particular, it assumes that the basis of implementable operations includes
+    any desired ideal operation followed by a depolarizing channel (meaning
+    that all :math:`\mathcal{O}_{\alpha} = \mathcal{D} \circ \mathcal{U}`,
+    where :math:`\mathcal{D}(\rho) =  (1 - \epsilon) \rho + \epsilon I/(2^n)`).
```

Comment:
Is this function buggy if `ideal_operation` has more than one operation in the circuit, then?

Reply:
Same question for `local_depolarizing_representation` below.

Reply:
I added a note on the docstring about this question. I hope it makes sense.

Reply:
Makes sense!